gtk/src/light/gtk-3.20/_drawing.scss

```diff
@@ -274,22 +274,18 @@
 
     $_bg: null;
     $_box_shadow: null;
-    $_border_top_color: null;
 
     @if $_headerbar {
       $_bg: lighten($c, 10%);
       $_border: $_bg;
-      $_border_top_color: $_bg;
       $_box_shadow: none;
     } @else if $c == $button_bg_color {
       $_bg: darken($c, if($variant=='light', 7%, 10%));
       $_border: $borders_color;
-      $_border_top_color: darken($_border, if($variant=='light', 25%, 8%));
       $_box_shadow: inset 0 1px 1px 0px rgba(0, 0, 0, 0.05);
```

Comment:
I think that box-shadow is part of the old pressed effect, have you tried to remove it?

Reply:
no Ithought that it looks very subtle and soft, I wont mind to leave it as it is but Ican understand if you want to remove it due to consistency

Reply:
I looked at that, because it seems that the pressed effect is still there, even if reduced

Reply:
Done. Looks even better, you are right! :)